strax/xenon/rundb.py

```diff
@@ -0,0 +1,73 @@
+import os
+import re
+import socket
+
+import pymongo
+
+import strax
+
+export, __all__ = strax.exporter()
+
+
+@export
+class RunDB(strax.StorageFrontend):
+    """Frontend that searches RunDB MongoDB for data.
+
+    Loads appropriate backends ranging from Files to S3.
```

Comment:
Maybe add the caveats about writing you wrote in the PR here. If I remember correctly, writing is supported, but it doesn't get registered back in the rundb.

Also perhaps worth noting the chosen write backend is files if you're on dali and S3 otherwise (right?).